app/controllers/items.js

```diff
@@ -2,112 +2,114 @@
   Items Controller
 */
 
-//3rd party modules
-var express = require('express');
+// 3rd party modules
 var mongoose = require('mongoose');
 var winston = require('winston');
 
-//own modules
+// own modules
 var DefaultController = require('./');
 
 var Item = mongoose.model('Item');
 var defaultCtrl;
 
-var Controller = function(){
-  defaultCtrl = new DefaultController(Item, 'Item');
-  
-  // Define route params
-  this.paramFormat = defaultCtrl.format();
-  this.paramItem = defaultCtrl.modelParam();
-
-  // Define handlers for rest calls
-  this.find = defaultCtrl.find;
-  this.create = customCreate;
-  
-  this.get = defaultCtrl.get;
-  this.update = customUpdate;
-  this.remove = customRemove; // Default one does not run pre-remove hook
-
-  this.getImage = getImage;
-
-  return this;
-}
-
 function customCreate(req, res) {
   var item = new Item(req.body);
-  item.save( function(err){
-    if(err) {
-	  winston.error(err.message); 
-	  return res.status(400).json({ error:err.message }); 
-	}
-        
-    if(res){
+  item.save((err) => {
+    if (err) {
+      winston.error(err.message);
+      return res.status(400).json({ error: err.message });
+    }
+
+    if (res) {
       return res.status(200).json(item);
     }
-  });  
-}
 
-function customUpdate(req, res) {
-  // Handle requests that only provide available or in_stock in a special manner
-  if (req.body.in_stock !== undefined && req.body.available === undefined) {
-    handleUpdateInStock(req, res);
-  }
-  else if (req.body.in_stock === undefined && req.body.available !== undefined) {
-	handleUpdateAvailable(req, res);
-  }
-  
-  // Updating the item body
-  for (var key in req.body) {
-	req.Item[key] = req.body[key];
-  }  
-  
-  // Regular save
-  req.Item.save(function(err) {
-	if (err) { 
-	  winston.error(err.message);
-	  return res.status(400).json({error:err.message}); 
-	}
-	else {
-	  res.status(200).json(req.Loan); 
-	}
+    return undefined;
```

Comment:
Serves no functional purpose, but EsLint enforces a rule which states that if a function returns anything, it must return something in all cases.